### PR TITLE
fix cpu communicator for omp

### PR DESCRIPTION
**Pull request by eddy16112**

### CODE REVIEW

**Comment by rohany** [comment]
Change looks fine to me, but I defer to @magnatelee for the final review.

File: legate/core/communicator.py [hunk 1]
```diff
@@ -120,7 +120,16 @@ def __init__(self, runtime: Runtime) -> None:
         )
         self._init_cpucoll = library.LEGATE_CORE_INIT_CPUCOLL_TASK_ID
         self._finalize_cpucoll = library.LEGATE_CORE_FINALIZE_CPUCOLL_TASK_ID
-        self._tag = library.LEGATE_CPU_VARIANT
+        num_omps = int(
```
Comment (magnatelee) [inline legate/core/communicator.py:123]: let's move this to `Runtime` and make the communicator use it, as I think it's better to consolidate all tunable lookups in one place for code maintenance.

**Comment by magnatelee** [comment]
left a minor change request. otherwise looks good.

File: legate/core/communicator.py (hunk 1 above)
Comment (eddy16112) [reply legate/core/communicator.py]: I have pushes a new commit for it. Besides omps, I also add tunable lookups for CPUs and GPUs.

**Review by magnatelee — APPROVED** [review]

**Pull request merged**